foyer/opls_validation/dimethyldisulfide/dimethyldisulfide.gro

```diff
@@ -0,0 +1,103 @@
+GROningen MAchine for Chemical Simulation
+  100
```

Comment:
Can you update all these gro files to only have 1 molecule each?

Reply:
Sure I'll do it now.